scipy/stats/_continuous_distns.py

```diff
@@ -1411,8 +1411,17 @@ def _stats(self, df):
         return mu, mu2, g1, g2
 
     def _entropy(self, df):
-        return (sc.gammaln(.5 * df) +
-                .5 * (df - np.log(2) - (df - 1) * sc.digamma(.5 * df)))
+
+        def regular_formula(df):
+            return (sc.gammaln(.5 * df) + 0.5 *
+                    (df - np.log(2) - (df - 1) * sc.digamma(0.5 * df)))
```

Comment:
```suggestion
            return (sc.gammaln(.5 * df)
                    + 0.5 * (df - np.log(2) - (df - 1) * sc.digamma(0.5 * df)))
```
https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator